test/e2e/channel_single_event_test.go

```diff
@@ -26,9 +26,9 @@ import (
 )
 
 func TestSingleBinaryEventForChannel(t *testing.T) {
-	helpers.SingleEventForChannelTestHelper(t, cloudevents.Binary, "v1alpha1", channelTestRunner)
+	helpers.SingleEventForChannelTestHelper(t, cloudevents.Binary, "v1alpha1", "messaging.knative.dev/v1alpha1", channelTestRunner)
```

Comment:
zomg, sorry for breaking. If you're not overriding anything (like here), you can just pass in ""

Reply:
fixed in e350562